cli/base-command.ts

```diff
@@ -318,5 +320,14 @@ export abstract class BaseCommand extends Command {
       estimatedGasUsed: estimatedGasUsed.toString(),
       gasPriceWei: gasPriceWei.toString(),
     });
+    const hops = routeAmounts[0]?.poolAddresses.length;
+    this.logger.info(`Total hops ${hops}`);
+
+    if (routeAmounts[0]?.protocol == Protocol.V3) {
+      const v3Route = routeAmounts[0] as V3RouteWithValidQuote;
+      this.logger.info(
+        `Total initialized ticks crossed ${v3Route.initializedTicksCrossedList.length}`
+      );
```

Comment:
Each element in the initializedTIcksCrossedList is the number of ticks crossed for each pool so you need to sum them

Reply:
Right! Actually these can just be deleted - I just needed these data points when I was manual testing with arbirtrum and optimism.

If you think these are good to print out though in general, I can change it back to summing over all the routes.